From aca2e7be7262e2918d0c666269e537325164ac2e Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 22 Oct 2014 18:12:39 -0400
Subject: [pci] Export MSI message relevant functions

Message-id: <20141022181239.10432.85519.stgit@amt.stowe>
Patchwork-id: 98078
O-Subject: [RHEL7.1 PATCH] PCI: Export MSI message relevant functions
Bugzilla: 1123366
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1123366
Upstream Status: 3b307ffe3faad60aeda0e9a4f661d5c1edbd761e
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=8150323
Testing: N/A

commit 3b307ffe3faad60aeda0e9a4f661d5c1edbd761e
Author: Gavin Shan <gwshan@linux.vnet.ibm.com>
Date:   Mon Sep 29 10:13:46 2014 -0600

    PCI: Export MSI message relevant functions

    The patch exports 2 MSI message relevant functions, which will be
    used by VFIO PCI driver. The VFIO PCI driver would be built as
    a module.

    Signed-off-by: Gavin Shan <gwshan@linux.vnet.ibm.com>
    Acked-by: Bjorn Helgaas <bhelgaas@google.com>
    Signed-off-by: Alex Williamson <alex.williamson@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/pci/msi.c b/drivers/pci/msi.c
index 7d2592d..9958639 100644
--- a/drivers/pci/msi.c
+++ b/drivers/pci/msi.c
@@ -344,6 +344,7 @@ void get_cached_msi_msg(unsigned int irq, struct msi_msg *msg)
 
  __get_cached_msi_msg(entry, msg);
 }
+EXPORT_SYMBOL_GPL(get_cached_msi_msg);
 
 void __write_msi_msg(struct msi_desc *entry, struct msi_msg *msg)
 {
@@ -388,6 +389,7 @@ void write_msi_msg(unsigned int irq, struct msi_msg *msg)
 
  __write_msi_msg(entry, msg);
 }
+EXPORT_SYMBOL_GPL(write_msi_msg);
 
 static void free_msi_irqs(struct pci_dev *dev)
 {
-- 
1.7.1